server.ts

```diff
@@ -36,7 +34,7 @@ const port = process.env.SERVER_PORT;
 
 const start = async (): Promise<void> => {
   try {
-    await db.sequelize.sync();
+    await db.sequelize.sync({ alert: true });
```

Comment:
I prefer to remove {alter:true} after our changes are applied to avoid any undesirable changes on our database by default